activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -387,6 +387,23 @@ def order!(*args) # :nodoc:
       self
     end
 
+    # Allows to specify an order by a specific set of values. Depending on your
```

Comment:
```suggestion
    # Allows specifying order by a specific set of values. Depending on your
```

Reply:
Yeah, I thought that language was weird too, but it's consistent with 4 other methods so I left it in.